lib/db/dbmi_base/login.c

```diff
@@ -297,10 +297,10 @@ static int get_login(const char *driver, const char *database, const char **user
 
     G_debug(3, "db_get_login(): drv=[%s] db=[%s]", driver, database);
 
-    user[0] = '\0';
-    password[0] = '\0';
-    host[0] = '\0';
-    port[0] = '\0';
+    user[0] = NULL;
```

Comment:
```suggestion
    *user = NULL;
    *password = NULL;
    *host = NULL;
    *port = NULL;
```
These are pointers to pointers, see code below.

Reply:
Updated as suggested.